### ISSUE TITLE
Secondary Nav Accordion Navigation - Close Screen Reader Focus

### ISSUE THREAD

**Issue by sherakama** [SU-SWS/decanter#606]
ID
1065666465
Violation
Ensure content that is intended to be hidden from all users is not rendered by assistive technology
Module Name
Multi Menu
Description
[Issue]
This issue impacts the accordion style menus.

When navigating with JAWS, the sub-menus can easily be opened with JAWS controls, and navigated through. The use of the escape key closes the menu and navigation continues as normal.

However, after pressing the escape key, the menu is not fully closed for the screen reader even though it visually appears to be. JAWS can still navigate into the submenu before or after it. The link state for aria-expanded is being changed appropriately and does state that the menu is no longer expanded.

If another menu item is opened via JAWS (or other controls such a mouse), the previously opened sub-menu is no longer accidentally available. However, the new opened sub-menu will have the same issue.

[User Impact]
Assistive technology such as screen readers may render hidden or unavailable content to the user if the content is not marked in a certain manner as "hidden" or not restricted from their navigation cycle. Having the wrong content rendered may cause users to read incorrect information or even activate links that are off screen. This can lead to confusion and incorrect results.

[Code Reference]
<ul class="su-multi-menu_menu-lv2 su-multi-menu_menu" id="su-acs-1-1" aria-labelledby="su-acc-1-1" >
<li class="su-multi-menu_item"><a href="https://www.stanford.edu" class="su-multi-menu_link" target="_blank">Stanford</a></li>
<li class="su-multi-menu_item"><a href="https://uit.stanford.edu/sws" class="su-multi-menu_link" target="_blank">Stanford Web Services</a></li>
<li class="su-multi-menu_item"><a href="https://ucomm.stanford.edu/" class="su-multi-menu_link">University Communications</a></li>
<li class="su-multi-menu_item"><a href="/section-components.html" class="su-multi-menu_link">Decanter components</a></li>
</ul>
Note
[Recommendation]
Ensure that when the menu is closed by other controls such as navigating out or with the escape key that the menu returns to a display:none; state. Test the submenu with a screen reader to ensure proper opening/closing.
Line Number
-1
Instance Severity
3
Thumbnail
